code/modules/reagents/recipes/recipes.dm

```diff
@@ -363,6 +363,13 @@
 	required_reagents = list("mindbreaker" = 1, "acetone" = 1, "inaprovaline" = 1)
 	result_amount = 3
 
+/datum/chemical_reaction/lean
+	result = "lean"
+	required_reagents = list("oxycodone" = 1, "chloralhydrate" = 1, "stoxin" = 1, "sodawater" = 1)
```

Comment:
list("tramadol" = 1, "chloralhydrate" = 1, "stoxin" = 1, "sodawater" = 1)


lean shouldn't require oxyco, and this has a lower painkiller strength than oxyco